src/components/Process/Header.tsx

```diff
@@ -18,6 +18,7 @@ import { CreatedBy } from './CreatedBy'
 import { ProcessDate } from './Date'
 import { ShareModalButton } from '~components/Share'
 import { useEffect, useState } from 'react'
+import { StampIcon } from '~components/ProcessCreate/Census/Gitcoin/StampIcon'
```

Comment:
Wrong import order. Still without the IDE feature to organize imports enabled? 👀